ui-lib/src/main/java/co/electriccoin/zcash/ui/screen/warning/AndroidNotEnoughSpace.kt

```diff
@@ -16,10 +17,11 @@ fun MainActivity.WrapNotEnoughSpace() {
     WrapNotEnoughSpace(this)
 }
 
+@OptIn(ExperimentalLifecycleComposeApi::class)
 @Composable
 private fun WrapNotEnoughSpace(activity: ComponentActivity) {
     val storageCheckViewModel by activity.viewModels<StorageCheckViewModel>()
-    val spaceRequiredToContinue by storageCheckViewModel.spaceRequiredToContinueMegabytes.collectAsState()
+    val spaceRequiredToContinue by storageCheckViewModel.spaceRequiredToContinueMegabytes.collectAsStateWithLifecycle()
```

Comment:
@HonzaR This might partially fix https://github.com/zcash/secant-android-wallet/issues/683

Reply:
Good job! It helped.